src/python/pants/backend/project_info/tasks/dependencies.py

```diff
@@ -25,9 +26,15 @@ def register_options(cls, register):
     register('--internal-only', type=bool,
              help='Specifies that only internal dependencies should be included in the graph '
                   'output (no external jars).')
-    register('--external-only', type=bool,
-             help='Specifies that only external dependencies should be included in the graph '
-                  'output (only external jars).')
+    register(
+      '--external-only',
+      type=bool,
+      help='Specifies that only external dependencies should be included in the graph output (only external jars).',
+      removal_version="1.26.0.dev1",
+      removal_hint="This feature is being removed. If you depend on this functionality, please let us know in the "
+                   "#general channel on slack. \nYou can join the pants slack here: "
+                   "https://pantsslack.herokuapp.com/ ",
```

Comment:
Can simplify to:

> on Slack at https://pantsbuild.slack.com/.

Note also the link. I took that from the documentation here: https://www.pantsbuild.org/howto_contribute.html#join-the-conversation

Reply:
From https://pantsbuild.slack.com/ you can only create an account if you have a foursquare/twitter/etc email. The herokuapp site allows anyone to invite themselves to pantsbuild slack. 

I figure anyone who is already a member of pants slack knows how to log in, and anyone who isn't a member needs to know how to join. 

Alternatively I could link to  https://www.pantsbuild.org/community.html